extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/NonJpaModelBuildItem.java

```diff
@@ -1,21 +0,0 @@
-package io.quarkus.hibernate.orm.deployment;
-
-import io.quarkus.builder.item.MultiBuildItem;
-
-/**
- * Model class that JPA should ignore
- *
- * @author Stéphane Épardaud
- */
-public final class NonJpaModelBuildItem extends MultiBuildItem {
```

Comment:
@FroMage I do remember you had to introduce this, but I don't remember why... seems like Yoann is right and this is dead code now, seems safe to remove. Are we missing something?

Reply:
@Sanne It's worse than "dead code now", from what I can see in the git log it has always been dead code.

Reply:
Safe to remove. It was made for when Reactive Panache had its own JPA-lite implementation.

Reply:
It was not dead code, though, it was used in a branch.

Reply:
But that branch is now dead.